app/controllers/saved_pages_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SavedPagesController < ApplicationController
+  include AuthenticatedApiConcern
+
+  before_action :set_user
+  # GET /api/saved_pages
+  def index
+    @saved_pages = SavedPage.where(oidc_user_id: @user.id)
+    render_api_response(saved_pages: present_pages(@saved_pages))
+  end
+
+private
+
+  def present_pages(saved_pages)
+    saved_pages.map { |saved_page| present_page(saved_page) }
+  end
+
+  def present_page(saved_page)
+    return {} if saved_page.nil?
+
+    { "page_path" => saved_page.page_path }
+  end
+  def set_user
+    @user = OidcUser.find_by(sub: @govuk_account_session.user_id)
+  end
```

Comment:
If we change this to
```
def user
  @user ||= OidcUser.find_by(sub: @govuk_account_session.user_id)
end
```
we can remove the `before_action` and we can use `user` instead of `@user`.

Reply:
that works well, I'll fix that up!